spinalcordtoolbox/deepseg/models.py

```diff
@@ -237,7 +283,45 @@
                              'and Dimitry Van De Ville of EPFL, with the files consisting of lumbar T2w scans (and '
                              'manual spinal cord segmentations) of 11 healthy (non-pathological) patients.',
          'url': 'https://github.com/ivadomed/lumbar_seg_EPFL',
-         'models': ['model_seg_epfl_t2w_lumbar_sc']}
+         'models': ['model_seg_epfl_t2w_lumbar_sc']},
+    'seg_sc_contrast_agnostic':
+        {'description': 'Spinal cord segmentation that is agnostic to contrast',
+         'long_description': 'This segmentation model for contrast agnostic spinal cord segmentation uses an NNUnet '
+                             'architecture, and was created with the MONAI package. Training data was taken from the '
+                             'Spine Generic Multi Subject dataset, with the 6 different contrasts used spanning 267 '
+                             'different healthy (non-pathological) patients.',
+         'url': 'https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/',
+         'models': ['model_seg_sc_contrast_agnostic_softseg_nnunet']},
+    'seg_sc_lesion_t2w_sci':
+        {'description': 'Traumatic SCI spinal cord/lesion seg for T2w contrast',
+         'long_description': 'This segmentation model for traumatic spinal cord injury segmentation uses an NNUnet '
+                             'architecture, and was created with the nnUNetV2 package. It is a multiclass model, '
+                             'outputting segmentations for both the spinal cord and SCI lesions. Training data '
+                             'consists of T2w scans from 191 SCI patients, spanning numerous resolutions and '
+                             'orientations, as well as multiple scanner manufacturers and field strengths. Training '
+                             'data was provided by the Balgrist Center at the University of Zurich.',
```

Comment:
```suggestion
         'long_description': 'This segmentation model for spinal cord injury segmentation uses a 3D U-Net '
                             'architecture, and was created with the nnUNetV2 framework. It is a multiclass model, '
                             'outputting segmentations for both the hyperintense SCI lesions and spinal cord. Training data '
                             'consists of T2w images (n=196), spanning numerous resolutions and '
                             'orientations, as well as multiple scanner manufacturers and field strengths'.,
```

Reply:
Slightly tweaking. @naga-karthik, do you agree?

Is it necessary to specify the sites that provided the training data here? This info will be available in the paper once it is published. If we decide to include the sites here, then besides Zurich, data was also from the University of Colorado School of Medicine, Aurora, Colorado and Paris.

Reply:
> Is it necessary to specify the sites that provided the training data here? This info will be available in the paper once it is published. If we decide to include the sites here, then besides Zurich, data was also from the University of Colorado School of Medicine, Aurora, Colorado and Paris.

Probably not! I imagine the specific attribution for the data is less relevant for the user, compared to details such as healthy vs. pathological, demographics, etc.). And, as you mention, the paper itself provides the site data.